worker/src/test/java/gov/cms/ab2d/worker/processor/ProgressTrackerIntegrationTest.java

```diff
@@ -97,23 +103,42 @@ void testIt() throws ExecutionException, InterruptedException {
         Bundle bundleA = BundleUtils.createBundle(entry1, entry2, entry3);
         Bundle bundleB = BundleUtils.createBundle(entry1, entry2, entry3, entry4);
 
-        when(bfdClient.requestPartDEnrolleesFromServer(contractId, 1)).thenReturn(bundleA);
-        when(bfdClient.requestPartDEnrolleesFromServer(contractId, 2)).thenReturn(bundleB);
+        when(bfdClient.requestPartDEnrolleesFromServer("C0001", 1)).thenReturn(bundleA);
+        when(bfdClient.requestPartDEnrolleesFromServer("C0001", 2)).thenReturn(bundleB);
 
         cut.processContractBenes(job, progressTracker);
 
         Job loadedVal = jobRepository.findById(job.getId()).get();
         assertEquals(30, loadedVal.getProgress());
     }
 
+    private User createUser() {
+
+        Random random = new Random();
```

Comment:
Why the usage of Random?  Randomness in tests can bite one badly, it tends to introduce non-deterministic behavior.

Reply:
Because that was the original behavior. I can remove it.

Reply:
That would be safer.